ext/thread_test/parallel_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'net/http'
+require 'json'
+require 'rspec/expectations'
+
+iterations = 100
+processes_per_catalog = 2
```

Comment:
so this file is the same as the other one, except that this one has multiple processes running in parallel? just making sure there weren't any other differences I missed.

Reply:
Yeah, pretty much. Duplicating the setup code gives you more freedom to tweak individual pieces of it, but it could probably be refactored.